apps/src/mixins.scss

```diff
@@ -31,3 +31,19 @@
     border: none !important;
   }
 }
+
+@mixin slider-thumb {
```

Comment:
Wondering if it makes sense to have these here, or to just keep in the music-play-view CSS file...typically I've seen shared mixins treated more like functions that take arguments to encapsulate shared bits of styling, but these seem pretty specific to the MusicPlayView? Alternatively I could see us creating a common slider component that uses these styles, but then we'd include these in the CSS module for that component rather than here. Another idea would be to create a mixin that handles all the variations between browsers internally and takes a few arguments such as color, height, etc.

Reply:
I can keep in the music-play-view CSS file for now then. I am going to maybe refactor the slider as a follow-up PR. Thanks!